### DISCUSSION TITLE
I need some help regarding the search result

### DISCUSSION THREAD

**Discussion by jess101** [microsoft-search/pnp-modern-search#1460] [Q&A]
I have deployed and used this pnp modern search webpart for one of our client that uses SharePoint online and we found out some this particular issue when a new user has been added to the site collection. It does not show all the files in the search webpart. I was thinking this might be a search crawl issue but I am not 100% sure on it. But what I notice is most of the files just recently added can be seen by the new account not everything or existing files in a document library.  If anyone knows or can enlighten me regarding search result that would be great.

**Comment by wobba** [comment]
If you add a user via an AAD group, or in a Team it should be instant. If adding as a user explicitly to the site or to a SharePoint access group then you need to wait for a "security crawl" which happens at unknow intervals.

So yes, expected.

**Reply by jess101** [reply]
Yes, that is what I was thinking as well. Because they did not use an AAD group and they just added the account directly to the site. I will verify this and hopefully this won't take days, that is what I was thinking as well. Because they did not use an AAD group and they just added the account directly to the site. I will verify this and hopefully this won't take days to be crawled then. Thanks for the info!

**Reply by wobba** [reply]
For this scenario you can go to site/library settings and find the "Re-index this library/site" button on one of the admin pages. That will ensure re-index is queued up when the crawler has capacity.

**Reply by jess101** [reply]
Thanks Mate! I have done everything reindex for library and site. But still not showing the correct search results. I think this is not related to the modern search webpart. It is a bit weird that this is only happening to newly added users in the site permission.
